src/yki/spec.clj

```diff
@@ -307,7 +307,10 @@
                                :user/street_address
                                :user/email]))
 
+(s/def ::is_strongly_identified boolean?)
+
 (s/def ::registration-init-response (s/keys :req-un [::exam_session
+                                                     ::is_strongly_identified
```

Comment:
Ei sinänsä täysin pakollinen muutos, koska tieto on pääteltävissä myös vastauksen `user` avaimen sisällöstä. Mielestäni kuitenkin frontendin kannalta fiksumpaa näin.